content/doc/book/pipeline/index.adoc

```diff
@@ -30,52 +30,255 @@ A _continuous delivery pipeline_ is an automated expression of your process for
-`Jenkinsfile`, is the same, it's generally considered best practice to define
+`Jenkinsfile` is the same, it is generally considered best practice to define
 the Pipeline in a `Jenkinsfile` and check that in to source control.
 
-Here's an example of a `Jenkinsfile`:
+
+=== Declarative versus Scripted Pipeline syntax
+
+A `Jenkinsfile` can be written using two types of syntax - Declarative and
+Scripted.
+
+Declarative and Scripted Pipelines are constructed fundamentally differently.
+Declarative Pipeline is a more recent feature of Jenkins Pipeline which:
+
+* provides richer syntactical features over Scripted Pipeline syntax, and
+* is designed to make writing and reading Pipeline code easier.
+
+Many of the individual syntactical components (or "steps") written into a
+`Jenkinsfile`, however, are common to both Declarative and Scripted Pipeline.
+Read more about how these two types of syntax differ in
+<<#pipeline-concepts,Pipeline concepts>> and
+<<#pipeline-syntax-overview,Pipeline syntax overview>> below.
+
+
+[[why]]
+== Why Pipeline?
+
+Jenkins is, fundamentally, an automation engine which supports a number of
+automation patterns. Pipeline adds a powerful set of automation tools onto
+Jenkins, supporting use cases that span from simple continuous integration to
+comprehensive continuous delivery pipelines. By modeling a series of related
+tasks, users can take advantage of the many features of Pipeline:
+
+* *Code*: Pipelines are implemented in code and typically checked into source
+  control, giving teams the ability to edit, review, and iterate upon their
+  delivery pipeline.
+* *Durable*: Pipelines can survive both planned and unplanned restarts of the
+  Jenkins master.
+* *Pausable*: Pipelines can optionally stop and wait for human input or approval
+  before continuing the Pipeline run.
+* *Versatile*: Pipelines support complex real-world continuous delivery
+  requirements, including the ability to fork/join, loop, and perform work in
+  parallel.
+* *Extensible*: The Pipeline plugin supports custom extensions to its DSL
+  footnoteref:[dsl] and multiple options for integration with other plugins.
+
+While Jenkins has always allowed rudimentary forms of chaining Freestyle Jobs
+together to perform sequential tasks,
+footnote:[Additional plugins have been used to implement complex behaviors
+utilizing Freestyle Jobs such as the Copy Artifact, Parameterized Trigger, and
+Promoted Builds plugins] Pipeline makes this concept a first-class citizen in
+Jenkins.
+
+Building on the core Jenkins value of extensibility, Pipeline is also extensible
+both by users with <<pipeline/shared-libraries#,Pipeline Shared Libraries>> and
+by plugin developers.
+footnoteref:[ghof,plugin:github-organization-folder[GitHub Organization Folder plugin]]
+
+The flowchart below is an example of one continuous delivery scenario easily
+modeled in Jenkins Pipeline:
+
+image:pipeline/realworld-pipeline-flow.png[alt="Pipeline Flow",width=100%]
```

Comment:
@gilesgas  This doesn't behave the same as 
`image::pipeline/realworld-pipeline-flow.png[title="Pipeline Flow", 800]` 

I understand your intent, but they are different and in this case the original should be used.